packages/clay-core/src/vertical-bar/VerticalBar.tsx

```diff
@@ -88,6 +112,9 @@ export function VerticalBar({
 
 	const id = useId();
 
+	const [internalPanelWidth, setInternalPanelWidth] =
+		React.useState(panelWidth);
```

Comment:
@pat270 you can use `useInternalState` here to make this property controlled, so developers can use this information to do anything else, your `panelWidth` and `onPanelWidthChange` will now solve both use cases.

Reply:
@matuzalemsteles I'm having trouble implementing this. Do we have any docs on how to use `useInternalState`? The error I'm dealing with right now is:
```
You provided a 'value' prop for a component without a handler 'onPanelWidthChange'. This will render the component with an internal state, if the component is to be uncontrolled, use 'undefined'. Otherwise, set the 'onPanelWidthChange' handler.
```
I really have no idea what I'm doing.

Reply:
I'll submit a commit implementing this.